crates/solvers-dto/Cargo.toml

```diff
@@ -6,10 +6,11 @@ edition = "2021"
 license = "MIT OR Apache-2.0"
 
 [dependencies]
+model = { path = "../model" }
```

Comment:
I don't know if this is the best way. But all the `Signature` type is huge, and it would be a lot of functions and structs to copy. Maybe having a crate for the `Signature`?